src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -34,6 +34,7 @@ sealed class Http2Stream : IDisposable
             private readonly HttpResponseMessage _response;
             private readonly HttpConnectionResponseContent _responseContent;
             private bool _disposed;
+            private bool _cancelled;
```

Comment:
Nit: cancel, canceling, canceler, canceled, cancellation... i.e. two Ls in cancellation, one L in everything else.  We're not always consistent on those spellings, but we try :)

Reply:
Even though I know this, I always instinctively add two 'l's. I'll go over this before the next commit :)